tensorflow/core/util/mkl_util.h

```diff
@@ -1260,8 +1260,8 @@ inline Status CreateBlockedMemDescHelper(const memory::dims& dim,
   DCHECK_EQ(dim.size(), strides.size());
 #ifdef ENABLE_MKLDNN_V1
   const int kNumDims = dim.size();
-  mkldnn_dim_t input_dims[kNumDims];
-  mkldnn_dim_t input_strides[kNumDims];
+  mkldnn_dim_t* input_dims = new mkldnn_dim_t[kNumDims];
```

Comment:
Could you please show the error message related to this fix? I can't find it in https://github.com/tensorflow/tensorflow/issues/37430.

Reply:
the file moved a little, the index aren't the same, but the log for the error is here .
line equivalence 1262 = 1252

```
ERROR: C:/tensorflow/tensorflow/core/kernels/BUILD:8128:1: C++ compilation of rule '//tensorflow/core/kernels:mkl_aggregate_ops' failed (Exit 2)
.\tensorflow/core/util/mkl_util.h(1253): error C2131: expression did not evaluate to a constant
.\tensorflow/core/util/mkl_util.h(1252): note: failure was caused by a read of a variable outside its lifetime
.\tensorflow/core/util/mkl_util.h(1252): note: see usage of 'dim'
.\tensorflow/core/util/mkl_util.h(1254): error C2131: expression did not evaluate to a constant
.\tensorflow/core/util/mkl_util.h(1252): note: failure was caused by a read of a variable outside its lifetime
.\tensorflow/core/util/mkl_util.h(1252): note: see usage of 'dim'
.\tensorflow/core/util/mkl_util.h(1256): error C3863: array type 'dnnl_dim_t [kNumDims]' is not assignable
.\tensorflow/core/util/mkl_util.h(1257): error C3863: array type 'dnnl_dim_t [kNumDims]' is not assignable
Target //tensorflow/tools/pip_package:build_pip_package failed to build
```

Reply:
I see. Thank you!